packages/terra-date-picker/tests/jest/__snapshots__/DateInput.test.jsx.snap

```diff
@@ -408,13 +408,17 @@ exports[`correctly applies the theme context className 1`] = `
         >
           <VisuallyHiddenText
             aria-live="off"
-            text=" Terra.datePicker.dateFormatLabel Terra.datePicker.dateFormat  "
+            text=" 
+            Terra.datePicker.dateFormatLabel 
+            Terra.datePicker.dateFormat. . Terra.datePicker.hotKey "
```

Comment:
Ideally we should only get one separator. it looks like there two separator being added here

Reply:
Fixed in https://github.com/cerner/terra-framework/pull/1799/commits/c3e67972b07cd602ddc27c2483d22e3512b31b16